c2corg_ui/static/js/documentediting.js

```diff
@@ -504,6 +504,22 @@ app.DocumentEditingController.prototype.pushToArray = function(object, property,
   app.utils.pushToArray(object, property, value, event);
 };
 
+/**
+ * @param {number} value
+ * @export
+ */
+app.DocumentEditingController.prototype.convertExposureTime = function(value) {
+  if (value === 0) {
+    this['scope']['image']['exposure_time_converted'] = this.alerts.gettext('cannot be 0');
```

Comment:
i used `aletrs.gettext` only to get a translated version of the message string t:P  but i'm not sure if that service does it like the `mainController's` `translate()`